flake/default.nix

```
@@ -24,7 +25,7 @@
   # See https://flake.parts/options/flake-parts-partitions.html
   partitions = {
     dev = {
-      module = ./dev;
+      module = builtins.trace "Evaluating dev partition" ./dev;
```

This should be for local testing only.


```suggestion
      module = ./dev;
```

...Maybe it would be worth adding this trace, but if so we should probably debate that in a dedicated PR?